File: Python/Assignment/Module – 2 (Fundamentals of python)/Task6.py
# 6)Write python program that swap two number with temp variable and without temp variable.
#Using Third varible
a = int (input("Enter First number: ")) #input forst number
b = int (input("Enter Second number: "))   #input second number
temp=a      #taking third variable to store values
a=b
b=temp
print("After swapping a:",a)    #swapping
print("After swapping b:",b)    #swapping

#Without using third variable
x = int (input("Enter First number: ")) #input forst number
y = int (input("Enter Second number: "))   #input second number
x,y = y,x
print("After swapping a:",x)    #swapping
print("After swapping b:",y)    #swapping

File: C(Practice)/task6.c
//swapping using third variable

#include<stdio.h>
main(){
	int a, b, temp;
	printf("Enter the value of a : ");
	scanf("%d",&a);
	printf("Enter the value of b : ");
	scanf("%d",&b);
	temp = a;
	a = b;
	b = temp;
	printf("After Swapping the value of A : %d",a);
	printf("\nAfter Swapping the value of B : %d",b);
}
